src/Game.js

```diff
@@ -17,11 +17,12 @@ class Game {
     this.roundCounter = 0;
     this.round = this.startNewRound();
     this.puzzleSlayer = null;
+    this.round;
   }
 
   startGame() {
     this.addPlayers();
-
+    this.round = this.startNewRound();
```

Comment:
Now that `startNewRound` is being called in `startGame`, is `this.round = this.startNewRound()` still needed in the constructor?